### PR TITLE
Add section calling out realtime streams

### PR DESCRIPTION
**Pull request by jbeemster**

### CODE REVIEW

File: docs/getting-started-on-community-edition/quick-start/index.md [hunk 1]
```diff
@@ -87,6 +87,24 @@ The sections below will guide you through setting up your destination to receive
...
+
+As part of the deployment, your data will be available in real-time streams corresponding to the cloud provider you have chosen. You can consume data directly from these streams, either in addition to or instead of the data warehouse.
+
+| Stream | AWS | GCP | Azure |
+|:----------|:---:|:---:|:-----:|
+| Kinesis | :white_check_mark: | :x: | :x: |
+| Pub/Sub | :x: | :white_check_mark: |:x: |
+| EventHubs | :x: | :x: | :white_check_mark: |
+
+For an out-of-the-box solution to accessing this data in real-time streams, you can [check out our Snowbridge project](/docs/destinations/forwarding-events/snowbridge/). Alternatively, if you want to develop a custom consumer, you can [leverage our Analytics SDKs](https://docs.snowplow.io/docs/destinations/analytics-sdk/) to parse the event formats more easily.
```
Comment (stanch) [inline docs/getting-started-on-community-edition/quick-start/index.md:100]:
```suggestion
For an out-of-the-box solution to accessing this data in real-time streams, you can [check out our Snowbridge project](/docs/destinations/forwarding-events/snowbridge/index.md). Alternatively, if you want to develop a custom consumer, you can [leverage our Analytics SDKs](/docs/destinations/analytics-sdk/index.md) to parse the event formats more easily.
```

**Review by stanch — APPROVED** [review]

File: docs/getting-started-on-community-edition/quick-start/index.md [hunk 2]
```diff
@@ -87,6 +87,24 @@ The sections below will guide you through setting up your destination to receive
 | BigQuery | — | :white_check_mark: | — |
 | Synapse Analytics | — | — | :white_check_mark: |
 
+## Real-Time Streaming Storage Options
```
Comment (alexanderdean) [inline docs/getting-started-on-community-edition/quick-start/index.md:90]: Should be "Real-time streaming storage options" (note the lower case letters) for consistency with other titles...

**Comment by alexanderdean** [comment]
Made one suggestion, otherwise LGTM!

File: docs/getting-started-on-community-edition/quick-start/index.md [hunk 3]
```diff
@@ -74,7 +74,7 @@ Details on how to configure the Azure Terraform Provider can be found [on the re
   </TabItem>
 </Tabs>
 
-## Storage options
+## Warehouse storage options
```
Comment (stanch) [inline docs/getting-started-on-community-edition/quick-start/index.md:77]:
```suggestion
## Storage options
```

File: docs/getting-started-on-community-edition/quick-start/index.md [hunk 4]
```diff
@@ -87,6 +87,24 @@ The sections below will guide you through setting up your destination to receive
 | BigQuery | — | :white_check_mark: | — |
 | Synapse Analytics | — | — | :white_check_mark: |
 
+## Real-time streaming storage options
```
Comment (stanch) [inline docs/getting-started-on-community-edition/quick-start/index.md:90]:
```suggestion
## Real-time streaming options
```

**Review by stanch — CHANGES_REQUESTED** [review]

**Pull request merged**